web lint SyntaxError: Unexpected token 'with'

### Self Checks

- [x] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).
- [x] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [x] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [x] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:)
- [x] Please do not modify this template :) and fill in all the required fields.

### Dify version

1.1.3

### Cloud or Self Hosted

Self Hosted (Docker)

### Steps to reproduce

1 clone
```
git clone git@github.com:langgenius/dify.git
```
2 cd web

3 pnpm install

4  pnpm run lint



```

> dify-web@1.1.3 lint /home/clark/idea/dify/web
> pnpm eslint --cache --cache-location node_modules/.cache/eslint/.eslint-cache


Oops! Something went wrong! :(

ESLint: 9.22.0

SyntaxError: Unexpected token 'with'
    at ModuleLoader.moduleStrategy (node:internal/modules/esm/translators:152:18)
    at ModuleLoader.moduleProvider (node:internal/modules/esm/loader:298:14)
```

### ✔️ Expected Behavior

_No response_

### ❌ Actual Behavior

_No response_

I can not reproduce this error in my local development environment.  Please check the dependencies.

```
> dify-web@1.1.3 lint /Users/crazywoola/Program/dify/web
> pnpm eslint --cache --cache-location node_modules/.cache/eslint/.eslint-cache

(node:43266) ExperimentalWarning: Importing JSON modules is an experimental feature and might change at any time
(Use `node --trace-warnings ...` to show where the warning was created)

/Users/crazywoola/Program/dify/web/app/(commonLayout)/app/(appDetailLayout)/[appId]/layout-main.tsx
  97:6  warning  React Hook useCallback has a missing dependency: 't'. Either include it or remove the dependency array  react-hooks/exhaustive-deps

/Users/crazywoola/Program/dify/web/app/(commonLayout)/datasets/(datasetDetailLayout)/[datasetId]/api/page.tsx
  5:15  warning  'props' is defined but never used. Allowed unused args must match /^_/u  unused-imports/no-unused-vars

/Users/crazywoola/Program/dify/web/app/(commonLayout)/datasets/(datasetDetailLayout)/[datasetId]/layout-main.tsx
  69:10  warning  'isShowTips' is assigned a value but never used. Allowed unused vars must match /^_/u  unused-imports/no-unused-vars
  69:32  warning  'toggleTips' is assigned a value but never used. Allowed unused vars must match /^_/u  unused-imports/no-unused-vars

/Users/crazywoola/Program/dify/web/app/(commonLayout)/datasets/DatasetCard.tsx
```

--- a/web/README.md
+++ b/web/README.md
@@ -7,7 +7,7 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
 ### Run by source code
 
 Before starting the web frontend service, please make sure the following environment is ready.
-- [Node.js](https://nodejs.org) >= v18.x
+- [Node.js](https://nodejs.org) >= v22.11.x
 - [pnpm](https://pnpm.io) v10.x
 
 First, install the dependencies:

--- a/web/package.json
+++ b/web/package.json
@@ -3,7 +3,7 @@
   "version": "1.2.0",
   "private": true,
   "engines": {
-    "node": ">=18.18.0"
+    "node": ">=v22.11.0"
   },
   "scripts": {
     "dev": "cross-env NODE_OPTIONS='--inspect' next dev",

